controllers/articles.controller.js

```diff
@@ -10,3 +11,13 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getArticles = (req, res, next) => {
+  fetchArticles()
+    .then((allArticles) => {
+      res.status(200).send({ allArticles });
+    })
+    .catch((err) => {
+      next(err);
+    });
```

Comment:
Good to see a catch block here in the controller to support with catching errors and then passing them to the middleware for error handling, well done!